changelogs/unreleased/gh-9399-raft-leader-steps-off-on-ER_WAL_IO.md

```diff
@@ -0,0 +1,7 @@
+## feature/raft
+
+* Now the leader resigns on the first encounter with `ER_WAL_IO` write error.
+  Immediately after this, the node tries to write a new term. If this attempt
```

Comment:
The just resigned node doesn't attempt to write a new term. It only does so if others bumped the term after its resign. Let's not mention this.

Same for the docbot request.

Reply:
Done.